nicoRDL/dotTheory-JS
# JS-202 (JS-352) JavaScript and the DOM: Getting a Handle on the DOM

## Related Ticket

https://codeandtheory.atlassian.net/browse/JS-352

## Description
- completed code challenges on query selection, selecting elements through the DOM
- completed all quizzes

## Steps to Validate
1. Open the index.html in the getElementbyId folder
2. Input a text and click the button
3. an Alert should appear

1. Open the index.html in the getElementByTagName folder
2. Observe the page
3. The Listed text should have different colors

1. Open the index.html in the querySelector folder
2. Open the dev tools console
3. One at a time, load the different variables (navigationLinks, galleryLinks, footerImages)
4. Each variable should have a different set of DOM elements

## Risks
N/A
